tests/apis/test_export.py

```diff
@@ -35,6 +36,16 @@ def test_export_moco(self):
         if stat != 0:
             print(output)
 
+    def test_export_yolox(self):
+        config_file = 'configs/detection/yolox/yolox_s_8xb16_300e_coco.py'
+        ori_ckpt = PRETRAINED_MODEL_YOLOXS_EXPORT
+        ckpt_path = f'{self.tmp_dir}/export_yolox_s_epoch300.pt'
+        stat, output = subprocess.getstatusoutput(
+            f'python tools/export.py {config_file} {ori_ckpt} {ckpt_path}')
+        self.assertTrue(stat == 0, 'export model failed')
+        if stat != 0:
```

Comment:
`if stat != 0` should be moved before self.assertTrue, otherwise the error log will not print if the assert fails